package-lock.json

```diff
@@ -2476,6 +2479,12 @@
       "integrity": "sha1-3F5pjL0HkmW8c+A3doGk5Og/YW4=",
       "dev": true
     },
+    "tr46": {
```

Comment:
What the heck is **`tr46`**? 🤷‍♂️  https://www.npmjs.com/package/tr46

![image](https://user-images.githubusercontent.com/194400/160355039-97cffef3-9b98-413e-8f45-5203cfc73d81.png)

41.3 million monthly downloads ... 📈 
Wonder why @dependabot (the parent module requiring `tr46`) is hard-coding version **`0.0.3`** when there's a **`v3.0.0`** ... 💭